### PR TITLE
Add support for Lumen

### PR DESCRIPTION
**Pull request by krenor**

### CODE REVIEW

**Comment by stefanzweifel** [comment]
Thanks for your PR. Will go through the code in the next days.
I've never really used Lumen and first have to setup a demo project to test the thing in both Lumen and Laravel (just want to be sure)

**Comment by stefanzweifel** [comment]
Looks fine for me.

Would be great if there would be a `is_laravel` or `is_lumen` helper. The changes made in af112f67c994bcf037c291d266be32c4ec9f1bbb, fba70c3aaf982cd9413b18f4c3d71210e6fd36d5 and e8eaab00c86a39cdd0dcf23878bc7ceb7514b9e4 are kinda ugly.

Even if I don't like comments that much, maybe it would be great to write some comments to see, that those changes have been made for Lumen compatibility.

**Comment by stefanzweifel** [comment]
I've tagged a new release: [1.7.0](https://github.com/stefanzweifel/laravel-stats/releases/tag/v1.7.0) 🎉

**Comment by krenor** [comment]
I used `$router` because in Lumen the [Application](https://github.com/laravel/lumen-framework/blob/5.6/src/Application.php#L22) does the routing. In Laravel the `Kernel` class resolves to the [Router](https://github.com/laravel/framework/blob/5.6/src/Illuminate/Routing/Router.php). It was mainly to indicate: "Hey, this thing does the routing". 

I agree about a helper to determine if running Lumen or Laravel (tho not through a function, because I don't want to polute the global functions). Although this check might be costly, therefore I'd have to figure out some sort of caching. I'll kinda figure this out next week and send another PR, aight?

**Comment by stefanzweifel** [comment]
Thanks for the clarification. I've never used Lumen before and didn't know there were such big differences.

Concerning the detecting if it's Lumen or Laravel: I think the current state is fine and adding such a function just adds more weight. I've added some declarative comments yesterday and I'm fine with that 🤷‍♂️ .

I've tried yesterday to come up with a way to check if an app is Larvel or Lumen: The easiest way is to check if `php artisan --version` contains the string "Lumen". But that feels a like a hack. I also would but that logic into a separate package, as I think other package developers could use that.

In summary: the current state of the package is fine for me. If you want to tackle that problem and you find a clean solution then go for it.
